python-checks/src/test/resources/checks/changeMethodContract/changeMethodContractIntermediate.py

```diff
@@ -0,0 +1,8 @@
+from changeMethodContractParent import ParentClass
+from typing import Any
+
+class IntermediateClass(ParentClass):
+  ...
+
+class IntermediateWithAnyClass(ParentClass[Any]):
```

Comment:
The real reason of the FN is not related to the `Any` type, but related to inheritance from types with generic type. So, I guess it will be easier to understand what's going on if the class name will be like `IntermediateWithGenericTypeParent` or something like that

Reply:
Makes sense!